### ISSUE TITLE
Fractal and Mandrebrot upgrades break theme templates

### ISSUE THREAD

**Issue by ethikz** [frctl/fractal#270]
I had Fractal 1.0.11, Handlebars 1.1.2 and Mandelbrot 1.0.7 and I upgraded both to the newest version, Fractal 1.1.4,  Handlebars 1.1.4 and Mandelbrot 1.1.0.  I didn't make any changes but it broke most of my template theme override files.

During the build process is where this happens.

```
✘ Failed to export url /components/detail/blockquote - (/project/node_modules/@frctl/mandelbrot/views/pages/components/detail.nunj)
  Template render error: (/project/node_modules/@frctl/mandelbrot/views/partials/pen/browser.nunj)
  Template render error: (/project/node_modules/@frctl/mandelbrot/views/partials/browser/browser.nunj)
  Template render error: (/project/theme-overrides/views/partials/browser/controls.nunj)
```

Are there breaking changes in each of these packages?

**Comment by allmarkedup** [comment]
Hey @ethikz - there are certainly no intentional breaking changes in those releases. Unfortunately with overrides it's possible that an unintentional  issue could have come up.  Have you managed to resolve the issue yet or is still a problem?

**Comment by ethikz** [comment]
@allmarkedup No I had to revert back to previous versions.

**Comment by dkhuntrods** [comment]
Hi @ethikz 
There was a recent change in Handlebars that effected partial block rendering.
Were you using this functionality by chance?

If so, I had to update from:
```
{{#if @partial-block}}
{{>@partial-block}}
{{/if}}
```
to 
```
{{#>@partial-block}}
{{/@partial-block}}
```

**Comment by ethikz** [comment]
@dkhuntrods 

Thanks for the response but no it broke the default templates, nothing anything custom I added.

**Comment by dkhuntrods** [comment]
@ethikz I'm guessing you've already tried the old 'blow away `node_modules` and re-install everything again' trick? If not, I'd suggest giving that a try?

**Comment by ethikz** [comment]
@dkhuntrods haha yep and clear npm cache and reinstall everything.

I'll try again and just report back with screenshots

**Comment by ethikz** [comment]
@dkhuntrods This is what I get

```
(lexicon-master/node_modules/@frctl/mandelbrot/views/pages/components/detail.nunj) Template render error: (lexicon-master/node_modules/@frctl/mandelbrot/views/partials/pen/browser.nunj) Template render error: (lexicon-master/node_modules/@frctl/mandelbrot/views/partials/browser/browser.nunj) Template render error: (lexicon-master/theme-overrides/views/partials/browser/controls.nunj) Error: Unable to call img["tabIcon"], which is undefined or falsey
```

I've remove `img[tabIcon]` and it works fine.

I'm curious how to debug this.  Is there a way to render in the console what the variable render as?

**Comment by dkhuntrods** [comment]
Ok, is `img.tabIcon(panel)` some sort of custom helper? I don't recognise that handlebars syntax.

**Comment by ethikz** [comment]
Lol that's a good question.  I'm not familiar with it either.  There is another dev working on this now that I have it up and running so I will check the them.  It worked before the upgrade though which is weird.  It looks as if it has something to do with the icons on the tabs where the code is.

**Comment by ethikz** [comment]
So I just checked previous versions of `controls.nunj` in mandelbrot and the change happened here

https://github.com/frctl/mandelbrot/commit/7df294f78675cfd06540d890b955c88076a9768a

**Comment by dkhuntrods** [comment]
@ethikz Unfortunately, this can't be the direct source, otherwise everyone on `v1.1.0` would be getting the same issue (which I can testify isn't the case :) ).
Are you by any chance using customised tabs or templates of some sort in your build?

**Comment by ethikz** [comment]
@dkhuntrods Yes. I referenced the issue I opened in Mandelbrot because of the customization.  There are some parts that we have customized like the panels but only in reference to adding a copy to clipboard button.

But because of the Mandelbrot change without release notes I didn't know what changes had happened so I had the older `panels.nunj` where the new one doesn't have the  `img["tabIcon"]` anymore :)
